src/utils/components/AddBootableVolumeModal/AddBootableVolumeModal.tsx

```diff
@@ -22,22 +23,30 @@ import {
   initialBootableVolumeState,
   RADIO_FORM_SELECTION,
 } from './utils/constants';
-import { createDataSource } from './utils/utils';
+import { createBootableVolume } from './utils/utils';
 
 import './AddBootableVolumeModal.scss';
 
 type AddBootableVolumeModalProps = {
   isOpen: boolean;
   onClose: () => void;
-} & AddBootableVolumeButtonProps;
+  onCreateVolume?: (source: BootableVolume) => void;
```

Comment:
this name is a bit misleading since it's not really creating a volume, maybe `onSelectCreatedVolume`?

Reply:
I think in this way the method is more generic. The method has nothing to do with the selection that happen in the catalog. WDYT?